aea/cli/init.py

```diff
@@ -65,10 +65,6 @@ def _registry_init(author):
 
             do_register(username, email, password, password_confirmation)
```

Comment:
Does registration imply login?

Reply:
Otherwise keep `do_login(username, password)`

Reply:
Yes, just found that is a part of registration functionality on Registry side. It signs in automatically on successful registration.